system/jlib/jptree.cpp

```diff
@@ -7905,6 +7905,14 @@ jlib_decl IPropertyTree * loadConfiguration(const char * defaultYaml, const char
             displayConfig(config, componentTag);
             exit(0);
         }
+#ifdef _DEBUG
+        else if (strsame(cur, "--hold"))
```

Comment:
This allows any component to be placed into "hold" mode at startup for ease of debugging (you can then connect to the pod manually using kubectl exec, and start the process under a debugger).